pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py

```diff
@@ -96,6 +95,8 @@ def validate_config(self, repo, config, config_conduit):
         :rtype: tuple of length two. Either (False, str) or (True, None)
         """
         path = config.get(constants.CONFIG_INSTALL_PATH)
+        if not path:
+            return False, _('config not specified')
```

Comment:
The error message should be clearer, something like: 
"An install_path has to be specified for the puppet install distributor."